### PR TITLE
Replace abnormal chinese words with normal ones

### PR DESCRIPTION
**Pull request by pzy2000**
Replace "通缉" with "悬赏", making chinese tokens able to display normal on yimmenu.
Add added popup info for FORCE_WANTED_LVL function, making the menu more understandable.

### CODE REVIEW

File: en_US.json [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
+    "Player Database": "Player Database",
```
Comment (Yimura) [inline en_US.json:2]: Remove this entry, I'll work on translating the rest of the menu later on.

**Review by Yimura — CHANGES_REQUESTED** [review]

File: en_US.json (hunk 1 above)
Comment (pzy2000) [reply en_US.json]: already removed, along with some more useless ones lol

File: zh_CN.json [hunk 2]
```diff
@@ -81,8 +82,8 @@
     "EXPLOSION_KARMA": "爆炸因果（反弹爆炸）",
     "DAMAGE_KARMA": "伤害因果（反弹伤害）",
     "FORCE": "强制",
-    "KILL_ALL": "杀死所有人",
-    "EXPLODE_ALL": "炸死所有人",
+    "KILL_ALL": "弄死所有人",
+    "EXPLODE_ALL": "炸飞所有人",
```
Comment (Aure7138) [inline zh_CN.json:86]: 弄死会不会过于"接地气"了一点

File: zh_CN.json [hunk 3]
```diff
@@ -251,7 +252,7 @@
     "SPAWN_FOR": "将他生成给",
     "CHANGE_PLAYER_MODEL": "更改玩家模型",
     "SPAWN_PED": "生成实体",
-    "SPAWN_BODYGUARD": "生成保镖",
+    "SPAWN_BODYGUARD": "生成安全卫士",
```
Comment (Aure7138) [inline zh_CN.json:255]: 保镖我认为在中文里，至少在简中里是最符合的翻译，安全卫士让我想到360安全卫士

Comment (pzy2000) [reply zh_CN.json:255]: 有道理，

File: zh_CN.json (hunk 2 above)
Comment (pzy2000) [reply zh_CN.json:86]: 当时改着玩的哈哈，其实都行

File: en_US.json [hunk 4]
```diff
@@ -310,6 +310,7 @@
     "FORCE_SHOW_HUD_ELEMENT": "Force show HUD element",
     "FORCE_SHOW_HUD": "Force show HUD",
     "FORCE_WANTED_LVL": "Force Wanted Level",
+    "FORCE_WANTED_LVL_INFO": "Force Others or Yourself into a specific Wanted Level",
```
Comment (Yimura) [inline en_US.json:313]: I'd change this text to `Keep a specific wanted level active on yourself.`

**Review by Yimura — CHANGES_REQUESTED** [review]

File: en_US.json (hunk 4 above)
Comment (pzy2000) [reply en_US.json:313]: I suppose you are right xD

**Review by Yimura — APPROVED** [review]

**Review by Yimura — APPROVED** [review]

**Pull request merged**